<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadCreationPageWizard</class>
 <widget class="QWizardPage" name="UploadCreationPageWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_integrate">
     <property name="text">
      <string>Integrate in database</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_upload">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_integrate">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_upload">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Datastore</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DatastoreComboBox" name="cbx_datastore"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatastoreComboBox</class>
   <extends>QComboBox</extends>
   <header>vectiler.gui.cbx_datastore</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
